EIPS/eip-2612.md

```diff
@@ -0,0 +1,204 @@
+---
+eip: 2612
+title: permit: 712-signed approvals
+author: Martin Lundfall (@Mrchico)
+discussions-to: https://github.com/ethereum/EIPs/issues/2613
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-04-13
+requires: 20, 712
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A function `permit` extending [ERC-20](https://eips.ethereum.org/EIPS/eip-20) which allows for approvals to be made via `secp256k1` signatures. This kind of "account abstraction for ERC-20" brings about two main benefits:
```

Comment:
Use relative path links with exact matches:
```
[ERC-20](./eip-20.md)
```

Reply:
Better to link to eips.ethereum.org as per @axic s comment https://github.com/ethereum/EIPs/pull/2612#discussion_r414488488

Reply:
I believe we have since discussed this and relative links are better as they work both in GitHub and on the EIPs site and any mirror sites as well.  cc @axic in case you still disagree or I am misremembering having a conversation somewhere with you about this.

Reply:
Only seen this ping now. We have since discussed and it seems the current recommendation is the absolute link, but perhaps this can be changed in the future.